index.js

```diff
@@ -11,93 +11,128 @@ const githubClient = new GithubClient(token);
 const slackClient = new SlackClient(webhook);
 
 async function doTheThing() {
-    let blocks = [{
-        type: 'section',
-        text: {
-            type: 'mrkdwn',
-            text: `:wave: GitHub Security Alerts Report\n\nThe following repositories have open vulnerability alerts and need your attention.`
-        }
-    }];
+  let blocks = [
+    {
+      type: "section",
+      text: {
+        type: "mrkdwn",
+        text: `:wave: GitHub Security Alerts Report\n\nThe following repositories have open vulnerability alerts and need your attention.`,
+      },
+    },
+  ];
 
-    const repos = await githubClient.getRepos(searchQuery);
+  const repos = await githubClient.getRepos(searchQuery);
 
-    await Promise.map(repos, async ({name, org}) => {
-        const alerts = await githubClient.getVulnerabilities(org, name);
-        const criticalAlerts = _.filter(alerts, { severity: 'critical', dismissed: false });
-        const highAlerts = _.filter(alerts, { severity: 'high', dismissed: false });
-        if (criticalAlerts.length > 0 || highAlerts.length > 0) {
-            blocks.push({ type: 'divider' });
-            blocks = blocks.concat(formatAlertsForSlack({ org, name, criticalAlerts, highAlerts }));
-        }
+  await Promise.map(repos, async ({ name, org }) => {
+    const alerts = await githubClient.getVulnerabilities(org, name);
+    const criticalAlerts = _.filter(alerts, {
+      severity: "critical",
+      dismissed: false,
     });
+    const highAlerts = _.filter(alerts, { severity: "high", dismissed: false });
+    if (criticalAlerts.length > 0 || highAlerts.length > 0) {
+      blocks.push({ type: "divider" });
+      blocks = blocks.concat(
+        formatAlertsForSlack({ org, name, criticalAlerts, highAlerts })
+      );
+    }
+  });
 
-    blocks.push({
-        type: 'context',
-        elements: [
-            {
-                type: 'mrkdwn',
-                text: `Query: ${searchQuery}`
-            }
-        ]
-    });
+  blocks.push({
+    type: "context",
+    elements: [
+      {
+        type: "mrkdwn",
+        text: `Query: ${searchQuery}`,
+      },
+    ],
+  });
 
-    if (process.env.POST_TO_SLACK === 'true') {
-        await slackClient.postMessage({ blocks });
-    } else {
-        console.log(`Slack blocks: ${JSON.stringify(blocks,null,2)}`)
+  if (process.env.POST_TO_SLACK === "true") {
+    const allBlocks = breakBlocks(blocks);
```

Comment:
this through line 56 are the primary change. `breakBlocks` returns an array of arrays of blocks. We post those one at a time in sequence